### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by chandra-rayaprol** [community/community#52518] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

 My github pages link,' [](https://raysapps.github.io/DeviceModels/)], is not working. I tried few suggested options but unfortunately it is still not working.
 
404
There isn't a GitHub Pages site here.

If you're trying to publish one, [read the full documentation](https://help.github.com/pages/) to learn how to set up GitHub Pages for your repository, organization, or user account.

[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)
could you please help in resolving this issue?

**Comment by airtower-luna** [comment]
The link is missing, but typically that means your site is missing an `index.html` file. Mind that URL paths are case sensitive, so e.g. `Index.html` instead of `index.html` will *not* work.

**Comment by chandra-rayaprol** [comment]
I changed the case and renamed the file from "Index.html" to "index.html." Unfortunately this has not resolved this issue and link still not working

**Comment by airtower-luna** [comment]
Can you link the actual repository? Without seeing it it's impossible to help other than guessing a common problem.

**Comment by chandra-rayaprol** [comment]
please check this  repository link: "https://github.com/Raysapps/DeviceModels"

**Comment by airtower-luna** [comment, marked as answer]
Take a look at the Actions tab of your repository, you can find the build logs there. For example, here's the latest one: https://github.com/Raysapps/DeviceModels/actions/runs/4669204708

The error message there sounds like Actions (which is used to deploy Pages sites) has an internal problem. The older runs seem to have the same problem, so you should probably [report the bug to GitHub support](https://support.github.com/contact/bug-report), us other users can't check their internal systems to see what's going on. Make sure to include the request ID listed in the error message.

**Reply by bd12930** [reply]
My build has been failing too since yesterday.

**Comment by chandra-rayaprol** [comment]
Thanks.  With the request ID, I reported this issue to Github support.
